main/test/ca/uwaterloo/flix/language/phase/TestParser.scala

```diff
@@ -318,4 +318,13 @@ class TestParser extends FunSuite with TestUtils {
     expectError[ParseError](result)
   }
 
+  test("ParseError.Enum.01") {
+    val input =
+      """
+        |enum Empty {}
```

Comment:
I wonder now if this should also be allowed? Why not?

Reply:
Both of these are allowed:

```
class Foo[a]
class Foo[a] {}
```

Reply:
Yeah, I misread the class stuff and thought empty braces were not allowed, so I tried to match.